controller-manager panic

```
I0127 06:50:36.191157       1 controller.go:109] ModelBooster controller started
I0127 06:50:36.191505       1 controller.go:113] ModelServing controller started
I0127 06:50:36.191513       1 controller.go:126] Autoscaler controller started
I0127 06:50:36.191519       1 controller.go:142] Started controllers without leader election
I0127 06:50:36.191591       1 reflector.go:358] "Starting reflector" type="*v1alpha1.AutoscalingPolicy" resyncPeriod="0s" reflector="pkg/mod/k8s.io/client-go@v0.34.2/tools/cache/reflector.go:290"
I0127 06:50:36.191615       1 reflector.go:404] "Listing and watching" type="*v1alpha1.AutoscalingPolicy" reflector="pkg/mod/k8s.io/client-go@v0.34.2/tools/cache/reflector.go:290"
I0127 06:50:36.191711       1 reflector.go:358] "Starting reflector" type="*v1alpha1.ModelServing" resyncPeriod="0s" reflector="pkg/mod/k8s.io/client-go@v0.34.2/tools/cache/reflector.go:290"
I0127 06:50:36.191726       1 reflector.go:404] "Listing and watching" type="*v1alpha1.ModelServing" reflector="pkg/mod/k8s.io/client-go@v0.34.2/tools/cache/reflector.go:290"
I0127 06:50:36.191764       1 reflector.go:358] "Starting reflector" type="*v1alpha1.AutoscalingPolicyBinding" resyncPeriod="0s" reflector="pkg/mod/k8s.io/client-go@v0.34.2/tools/cache/reflector.go:290"
I0127 06:50:36.191816       1 reflector.go:404] "Listing and watching" type="*v1alpha1.AutoscalingPolicyBinding" reflector="pkg/mod/k8s.io/client-go@v0.34.2/tools/cache/reflector.go:290"
I0127 06:50:36.191298       1 main.go:219] Starting webhook server on :8443
E0127 06:50:36.192129       1 panic.go:262] "Observed a panic" panic="runtime error: invalid memory address or nil pointer dereference" panicGoValue="\"invalid memory address or nil pointer dereference\"" stacktrace=<
        goroutine 32 [running]:
        k8s.io/apimachinery/pkg/util/runtime.logPanic({0x19e7f88, 0xc00047fbf0}, {0x1501b40, 0x23cd120})
                /go/pkg/mod/k8s.io/apimachinery@v0.34.2/pkg/util/runtime/runtime.go:132 +0xbc
        k8s.io/apimachinery/pkg/util/runtime.handleCrash({0x19e7f50, 0x2415000}, {0x1501b40, 0x23cd120}, {0x2415000, 0x0, 0x19eef80?})
                /go/pkg/mod/k8s.io/apimachinery@v0.34.2/pkg/util/runtime/runtime.go:107 +0x116
        k8s.io/apimachinery/pkg/util/runtime.HandleCrash({0x0, 0x0, 0xc0003dc000?})
                /go/pkg/mod/k8s.io/apimachinery@v0.34.2/pkg/util/runtime/runtime.go:64 +0x105
        panic({0x1501b40?, 0x23cd120?})
                /usr/local/go/src/runtime/panic.go:792 +0x132
        github.com/volcano-sh/kthena/pkg/model-serving-controller/controller.(*ModelServingController).Run(0xc00043e0f0, {0x19e7fc0, 0xc000419590}, 0x5)
                /workspace/pkg/model-serving-controller/controller/model_serving_controller.go:457 +0x241
        created by github.com/volcano-sh/kthena/pkg/controller.SetupController.func1 in goroutine 1
                /workspace/pkg/controller/controller.go:112 +0x1d0
 >
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
        panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x38 pc=0x12e3b41]

goroutine 32 [running]:
k8s.io/apimachinery/pkg/util/runtime.handleCrash({0x19e7f50, 0x2415000}, {0x1501b40, 0x23cd120}, {0x2415000, 0x0, 0x19eef80?})
        /go/pkg/mod/k8s.io/apimachinery@v0.34.2/pkg/util/runtime/runtime.go:114 +0x1a9
k8s.io/apimachinery/pkg/util/runtime.HandleCrash({0x0, 0x0, 0xc0003dc000?})
        /go/pkg/mod/k8s.io/apimachinery@v0.34.2/pkg/util/runtime/runtime.go:64 +0x105
panic({0x1501b40?, 0x23cd120?})
        /usr/local/go/src/runtime/panic.go:792 +0x132
github.com/volcano-sh/kthena/pkg/model-serving-controller/controller.(*ModelServingController).Run(0xc00043e0f0, {0x19e7fc0, 0xc000419590}, 0x5)
        /workspace/pkg/model-serving-controller/controller/model_serving_controller.go:457 +0x241
created by github.com/volcano-sh/kthena/pkg/controller.SetupController.func1 in goroutine 1
```

This is tested with latest image today without installing volcano

--- a/pkg/model-serving-controller/podgroupmanager/manager.go
+++ b/pkg/model-serving-controller/podgroupmanager/manager.go
@@ -82,7 +82,6 @@ func NewManager(kubeClient kubernetes.Interface, volcanoClient volcanoclient.Int
 			newManager.hasSubGroupPolicy.Store(false)
 		} else {
 			klog.Errorf("failed to get PodGroup CRD: %v", err)
-			return nil
 		}
 	} else {
 		newManager.handlePodGroupCRDChange(crd, false)

